A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -844,6 +848,40 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
         SimVar.SetSimVarValue("L:A320_Neo_ADIRS_ALIGN_LIGHT_3", "Bool", (SimVar.GetSimVarValue("L:A320_Neo_ADIRS_KNOB_3", "Enum") == 1 && ADIRSState != 2) );
 
     }
+
+    SetBalanceData(data) {
+        this._balanceData = data.balance;
+    }
+
+    async trySetZFWCG(input, useLbs = false) {
```

Comment:
```suggestion
    trySetZFWCG(input, useLbs = false) {
```

`async` does nothing here, since the entire UI is single-threaded.